sched/definitions.go

```diff
@@ -17,3 +17,33 @@ type Task struct {
 	Command    []string // Argv to run
 	SnapshotId string
 }
+
+type JobState int
+
+const (
+	// An unambiguous 0-value.
+	UNKNOWN JobState = iota
+	// Waiting to run.
+	PENDING
+	// Running
+	RUNNING
+	// Done
+	DONE
+)
+
+type JobStatus struct {
+	ID    string
+	State JobState
+}
+
+func PendingStatus(jobID string) (s JobStatus) {
+	s.ID = jobID
+	s.State = PENDING
+	return s
+}
+
+func UnknownStatus(jobID string) (s JobStatus) {
```

Comment:
Should there be RunningStatus & DoneStatus functions.  
Should we prefix these methods with Make since they are Factory Methods?